client/src/pages/Login/Login.test.js

```diff
@@ -1,9 +1,11 @@
 import { render, screen, fireEvent, waitFor } from '@testing-library/react';
-import App from './App';
-import * as request from './utils/makeRequest';
+
+import Login from './index';
+import App from '../../App';
```

Comment:
Do you mean to create unit or integration tests?
In case integration - everything is ok. But in case unit
1. `App` has features:
	- wrap everything with `<AppProvider />`
	- wrap everything with `<Router />`
	- render `<LoginPage />` for root path url (by default?)
2. `AppProvider` has features to provide a bunch of properties and methods through the `context`
3. `LoginPage` has a feature to render `<LoginFrom />`
4. `LoginFrom` has features:
	- render 2 inputs and a button
	- send a request by clicking on a button
	
So you need to split this test cases in different `.test.js` files.

Reply:
Tried to fix it in the next PR.